src/main/resources/public/ts/sniplets/content/workspace-nextcloud-toolbar.sniplet.ts

```diff
@@ -135,8 +135,12 @@ export class ToolbarSnipletViewModel implements IViewModel {
 
     deleteDocuments(): void {
         const paths: Array<string> = this.vm.selectedDocuments.map((selectedDocument: SyncDocument) => selectedDocument.path);
+        const nextcloudTreeController = this.vm.getNextcloudTreeController();
         this.vm.nextcloudService.deleteDocuments(model.me.userId, paths)
-            .then(() => {
+            .then(() => this.vm.nextcloudService.deleteTrash(model.me.userId))
```

Comment:
Here, we empty the trash after the file delete in order to anticipate the feature of displaying trashbin in ENT tree. We didn't want to include the trash delete in the delete flow as it's not part of the nextcloud server delete process.